hassio-google-drive-backup/backup/ui/uiserver.py

```diff
@@ -392,7 +392,7 @@ def content():
 
     async def token(self, request: Request) -> None:
         self._global_info.setIngoreErrorsForNow(True)
-        creds_deserialized = json.loads(str(base64.b64decode(request.query.get('creds').encode("utf-8")), 'utf-8'))
+        creds_deserialized = json.loads(str(base64.b64decode(request.query.get('creds').encode("utf-8")), 'utf-8')).replace(" ", "")

```

Comment:
Use the str.trim() method since that will just remove whitespace from the ends, and you'll need to modify the string getting returned form the request query rather than the json object its serialized into, eg:
```python
creds_deserialized = json.loads(str(base64.b64decode(request.query.get('creds').encode("utf-8").trim()), 'utf-8'))
```

Reply:
Done, thank you!